staging/src/k8s.io/pod-security-admission/test/fixtures.go

```diff
@@ -45,9 +45,21 @@ func init() {
 	// Define minimal valid restricted pods.
 	//
 
-	// 1.0+: baseline + runAsNonRoot=true
+	// 1.0+: baseline + runAsNonRoot=true + seccomp
 	restricted_1_0 := tweak(baseline_1_0, func(p *corev1.Pod) {
 		p.Spec.SecurityContext = &corev1.PodSecurityContext{RunAsNonRoot: pointer.BoolPtr(true)}
+
+		// seccomp annotation
+		p.Annotations = map[string]string{}
+		p.Annotations["seccomp.security.alpha.kubernetes.io/pod"] = corev1.SeccompProfileRuntimeDefault
+
+		// seccomp field
+		//
+		// Normally, it would make sense to add this at v1.19+. However, it is added here
+		// as tests will fail due to the "alpha annotation is deprecated" warning otherwise.
```

Comment:
we should fix the warning check and register this at 1.19+... it's more important to make sure pre-1.19 policies don't require the field to be set